base/settings/R/loadPath_sitePFT.R

```diff
@@ -0,0 +1,24 @@
+#' Title loadPath.sitePFT
+#'
+#' @param settings pecan setting list.
+#' @param Path  Character of file name with extension. The path will be generated using the outdir tag in pecan settings.
+#'
+#' @return a dataframe of two columns of site and pft
+#' @export loadPath.sitePFT
+#' @description The csv or the text file needs to have a header and be separated using comma. Under the first column in the text file, one needs to specify the site id and in the second column there has to be the name of the PFT.
```

Comment:
There's ambiguity here about how the system handles multi-PFT runs. Since you're strictly assuming that there are two columns then I assume you have to enter the site ID multiple times (e.g. a site with 12 PFTs would have 12 rows).

Reply:
That's correct. 
I'm hoping the process of creating the lookup table will be all automated by a series of functions like `loadPath.sitePFT` which makes this just a matter of formating the lookup table.